build.sh

```diff
@@ -10,4 +10,5 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
 done
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
-"$DIR/run.sh" build "$@"
```

Comment:
Can you please explain why this was necessary?

Reply:
Yes, when I test on WSL and docker, I got permission error to run eng/common/build.sh, not sure if that will also happen on CI so I add it anyway.

---
In reply to: [501871675](https://github.com/dotnet/machinelearning/pull/5423#discussion_r501871675) [](ancestors = 501871675)

Reply:
But is this still necessary if you run this once from a linux environment and check the updated file in?
Since the rest of Arcade doesn't have this, I am wondering if you just need to do it only once.

Also, why did we need to replace run.sh line with a direct call to eng/common/build.sh?

---
In reply to: [501875979](https://github.com/dotnet/machinelearning/pull/5423#discussion_r501875979) [](ancestors = 501875979,501871675)

Reply:
Yes, you are right. Seems git will also track file execute permission and we can check in the file permission change. At first I'm not able to see any change in file content in eng/common/build.sh after change this file's execute permission so I undo the change, I just tried "git diff" and it shows the permission change like below:

diff --git a/eng/common/build.sh b/eng/common/build.sh
old mode 100644
new mode 100755

run.sh is using the old build tools to build the project so I replaced with eng/common/build.sh which will build ML.NET with arcade just like what you did for windows build.


---
In reply to: [501886537](https://github.com/dotnet/machinelearning/pull/5423#discussion_r501886537) [](ancestors = 501886537,501875979,501871675)

Reply:
If run.sh is pointing to the old build tools, can we delete it? (And run.cmd?)

---
In reply to: [501910887](https://github.com/dotnet/machinelearning/pull/5423#discussion_r501910887) [](ancestors = 501910887,501886537,501875979,501871675)

Reply:
Yes, there are lots of build related files we can delete from old build tools. I'm thinking we can do a clean up at the end of our arcade work if case we need something or reference something from old build tools. 

---
In reply to: [501919713](https://github.com/dotnet/machinelearning/pull/5423#discussion_r501919713) [](ancestors = 501919713,501910887,501886537,501875979,501871675)

Reply:
Let us clean up as we along.

---
In reply to: [501935956](https://github.com/dotnet/machinelearning/pull/5423#discussion_r501935956) [](ancestors = 501935956,501919713,501910887,501886537,501875979,501871675)